sycamore/transforms/summarize.py

```diff
@@ -60,10 +61,10 @@ def summarize(self, document: Document) -> Document:
         return document
 
     def _summarize_text_element(self, element: Element) -> Element:
-        if self._llm.is_chat_mode:
-            prompt = TEXT_SUMMARIZER_GUIDANCE_PROMPT_CHAT
+        if self._llm.is_chat_mode():
+            prompt: GuidancePrompt = TextSummarizerGuidancePromptChat()
```

Comment:
is it too complex if we implemented each prompt as a class with get_prompt_, get_chat_prompt methods? Might be easier to compose prompts and use in places like these

Reply:
That's a good question. Let me play around it a bit. Totally on board if we can find a way to unify them. I also wonder if there is some way to extract basic prompts from specific LLMs. I know that won't work in all cases, but ideally if we wanted to try Anthropic or Mixtral for entity extraction, for instance, we wouldn't have to start from scratch.